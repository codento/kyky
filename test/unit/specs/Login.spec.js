import { shallow, createLocalVue } from 'vue-test-utils'
import View from '@/views/Login'

describe('Login.vue', () => {
  let localVue = createLocalVue()
  const cmp = shallow(View, {
    localVue
  })
  it('Template is correct', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
