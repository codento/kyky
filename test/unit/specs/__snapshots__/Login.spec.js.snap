// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login.vue Template is correct 1`] = `
<div
  class="container"
  style="padding: 30px;"
>
  <form>
    <input
      class="form-control"
      id="exampleFormControlInput1"
      placeholder="Käyttäjätunnus"
      type="username"
    />
     
    <input
      class="form-control"
      id="exampleFormControlInputPass"
      placeholder="Salasana"
      type="password"
    />
     
    <input
      class="btn btn-primary"
      disabled="disabled"
      type="submit"
      value="Kirjaudu"
    />
  </form>
   
  <!---->
</div>
`;
